### PR TITLE
[Patch] Linux support and pyproject.toml

### PR DESCRIPTION
**Pull request by ahmubashshir**
Changes:

284743e (Mubashshir, 60 seconds ago)
   add linux support and restructure source code.

895ab4d (Mubashshir, 2 minutes ago)
   use pyproject.toml instead of requirements.txt [pep-0518]

### CODE REVIEW

**Comment by m4hi2** [comment]
@ahmubashshir Thank you for your time and effort. 💚

File: routEar/__main__.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        '/org/freedesktop/Notifications'
+    )
+    notif_id = None
+else:
+    print("unsupported platform")
+    sys.exit(1)
+
+def notify( title='title', body='text', icon='python'):
+    if sys.platform == "win32":
+        toaster.show_toast(
```
Comment (m4hi2) [inline routEar/__main__.py]: @ahmubashshir "toaster" :P

Comment (ahmubashshir) [reply routEar/__main__.py]: oops

File: routEar/__main__.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            title,
+            body,
+            threaded=True,
+            icon_path=path.join(next(i for i in routEar.__path__),'data', '%s.ico'%icon)
```
Comment (m4hi2) [inline routEar/__main__.py]: @ahmubashshir This line is generating the following error: 
```
File "D:\Documents\PythonCodes\routerShit\routEar\__init__.py", line 60, in callback
    notify(message_line_1, message_line_2)
File "D:\Documents\PythonCodes\routerShit\routEar\__init__.py", line 29, in notify
    next(i for i in routEar.__path__), 'data', '%s.ico' % icon)
NameError: name 'routEar' is not defined
```

Comment (ahmubashshir) [reply routEar/__main__.py]: I forgot that __path__ was a global constant :3

File: routEar/__main__.py (hunk 1 above)
Comment (ahmubashshir) [reply routEar/__main__.py]: fixed in #3

File: routEar/__main__.py (hunk 2 above)
Comment (ahmubashshir) [reply routEar/__main__.py]: fixed in #3

**Pull request merged**